content/archive/nwerc.md

```diff
@@ -4,5 +4,13 @@ Title: NWERC Problem Set Archive
 menu: Archive
 ---
 # {.Title}
+The first European Regional Contest (ERC) was organised in 1983.<br>
+In 1992, the region was split in west and east,
+which created the Western European Regional Contest (WERC).<br>
+In 1994, the region was split again in north and south,
+which created the North Western European Regional Collegiate Programming Contest (NWERCPC),
```

Comment:
> The 1999 one might require more research, I don't got any data on those in the folders I have at home. But from 1995 it was NWERC for sure. See also the preface from the contest director of NWERC 95

It would be cool to have this preface linked in the intro paragraph of the NWERC archive as source reference :smile:

Reply:
done ✔️